ansible/roles/shd-client-core/defaults/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 SHD_CLIENT_IMAGE: quay.io/schulcloudverbund/shd-client
 SHD_CLIENT_PREFIX: superhero.
+WITH_SHD_CLIENT: false
```

Comment:
Put this at https://github.com/hpi-schul-cloud/dof_app_deploy/blob/main/ansible/group_vars/all/with.yml